.travis.yml

```diff
@@ -5,6 +5,8 @@ matrix:
         - php: 5.3.3
           env: 'COMPOSER_FLAGS="--prefer-stable --prefer-lowest"'
         - php: 5.3
+        - php: 5.3
```

Comment:
My intention here is this.

If we need to build the phar with `--prefer-stable --prefer-lowest` with PHP 5.3.29, we should test that combination.

I'd like to run a build both with or without `--prefer-stable --prefer-lowest`